modules/system/primary-user.nix

```diff
@@ -0,0 +1,60 @@
+{
+  lib,
+  options,
+  config,
+  ...
+}:
+
+{
+  options = {
+    system.primaryUser = lib.mkOption {
```

Comment:
Can we have an option to disable primary user, so that dogfooding finds problems with not having a primary user sooner?

Reply:
I just realized the default is `null`, so never mind.

Reply:
Yeah, leaving it at the default will ensure your system doesn’t depend on any primary user and give a hopefully‐helpful error message if you set anything that requires it.